logging/src/main/java/life/qbic/logging/api/Logger.java

```diff
@@ -0,0 +1,44 @@
+package life.qbic.logging.api;
+
+/**
+ * Simplistic interface of a {@link Logger}, that enables basic logging functionality for different
+ * log levels.
+ *
+ * @since 1.0.0
+ */
+public interface Logger {
```

Comment:
I am missing the inclusion of a throwable to debug. Further I did not see any access to the warn log level here. Is this intentional?